README.md

```diff
@@ -1,179 +1,12 @@
-#import <PowerAuth.h>
-#import <React/RCTRootView.h>
-
-@interface AppDelegate ()
-@property (nonatomic, strong) NSDictionary *launchOptions;
-@end
-
-@implementation AppDelegate
-
-- (RCTBridge *)initializeReactNativeApp
-{
-  RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:self.launchOptions];
-  
-  // POWERAUTH CONFIGURATION
-  PowerAuth *pa = [bridge moduleForClass:PowerAuth.class];
-  if (pa) {
-    PowerAuthConfiguration *config = [[PowerAuthConfiguration alloc] init];
-    config.instanceId = @"your-instance-id";
-    config.appKey = @"APPLICATION_KEY";
-    config.appSecret = @"APPLICATION_SECRET";
-    config.masterServerPublicKey = @"KEY_SERVER_MASTER_PUBLIC";
-    config.baseEndpointUrl = @"https://your-powerauth-endpoint.com/";
-    if(![pa configureWithConfig:config keychainConfig:nil clientConfig:nil]) {
-      NSLog(@"Failed to configure PowerAuth module");
-    }
-  } else {
-    NSLog(@"PowerAuth module not found");
-  }
-  
-  // CODE GENERATED BY THE REACT NATIVE TEMPLATE
-  
-  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"main" initialProperties:nil];
-  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
-
-  UIViewController *rootViewController = [UIViewController new];
-  rootViewController.view = rootView;
-  self.window.rootViewController = rootViewController;
-  [self.window makeKeyAndVisible];
-
-  return bridge;
-}
-
-- (void)appController:(EXUpdatesAppController *)appController didStartWithSuccess:(BOOL)success
-{
-  appController.bridge = [self initializeReactNativeApp];
-}
-
-@end
-```
-
-For more information about the native configuration, you can visit [official documentation of the native SDK](https://github.com/wultra/powerauth-mobile-sdk/blob/develop/docs/PowerAuth-SDK-for-iOS.md#configuration).
-
-## API reference
-
-For API reference, visit [folder with definition files](https://github.com/wultra/react-native-powerauth-mobile-sdk/blob/develop/lib/) where you can browse all documented available methods.
-
-> More detailed documentation will be added later. If you need any information regarding the status of this library, don't hesitate to [contact us](#contact).  
-> 
-> For information, you can visit the [native PowerAuth SDK for Mobile Apps](https://github.com/wultra/powerauth-mobile-sdk) as this library acts as a bridge between the Javascript environment and the native module.*
-
-## Demo application & example usage
-
-Demo application with the integration of the PowerAuth React Native SDK can be found inside the `demoapp` folder.
+## Documentation
 
-Visit [App.tsx](https://github.com/wultra/react-native-powerauth-mobile-sdk/blob/develop/demoapp/App.tsx) for example usage of every available API method.
+The documentation is available at the [Wultra Developer Portal](https://developers.wultra.com/products/mobile-security-suite/develop/react-native-powerauth-mobile-sdk/) or inside the [docs](docs) folder.
```

Comment:
The URL should point to the Github repository, the Docucheck should automatically pick it up (if not, let's report the issue).

Reply:
This file is not part of the documentation - it's just repo readme.